src/test/kotlin/io/github/mojira/arisa/modules/PrivacyModuleTest.kt

```diff
@@ -1,426 +1,526 @@
 package io.github.mojira.arisa.modules
 
+import io.github.mojira.arisa.domain.CommentOptions
 import io.github.mojira.arisa.utils.RIGHT_NOW
 import io.github.mojira.arisa.utils.mockAttachment
 import io.github.mojira.arisa.utils.mockChangeLogItem
 import io.github.mojira.arisa.utils.mockComment
 import io.github.mojira.arisa.utils.mockIssue
 import io.kotest.assertions.arrow.either.shouldBeLeft
 import io.kotest.assertions.arrow.either.shouldBeRight
-import io.kotest.core.spec.style.StringSpec
+import io.kotest.core.spec.style.FunSpec
 import io.kotest.matchers.shouldBe
 
-private val ALLOWED_REGEX = listOf("allowed@.*".toRegex())
-private val MODULE = PrivacyModule("message", "\n----\nRestricted by PrivacyModule ??[~arisabot]??", ALLOWED_REGEX, emptyList())
 private val TWO_SECONDS_AGO = RIGHT_NOW.minusSeconds(2)
 private val TEN_SECONDS_AGO = RIGHT_NOW.minusSeconds(10)
 
-class PrivacyModuleTest : StringSpec({
-    "should return OperationNotNeededModuleResponse when the ticket is marked as private" {
-        val issue = mockIssue(
-            securityLevel = "private"
+private const val MADE_PRIVATE_MESSAGE = "made-private"
+private const val COMMENT_NOTE = "\n----\nRestricted by PrivacyModule ??[~arisabot]??"
+
+private fun createModule(
+    message: String = MADE_PRIVATE_MESSAGE,
+    commentNote: String = COMMENT_NOTE,
+    allowedEmailRegexes: List<Regex> = emptyList(),
+    sensitiveTextRegexes: List<Regex> = emptyList(),
+    sensitiveFileNameRegexes: List<Regex> = emptyList()
+) = PrivacyModule(
+    message,
+    commentNote,
+    allowedEmailRegexes,
+    sensitiveTextRegexes,
+    sensitiveFileNameRegexes
+)
+
+class PrivacyModuleTest : FunSpec({
```

Comment:
Changed this from `StringSpec` to `FunSpec` to allow using Kotest's `context` for grouping the test methods. For `StringSpec` that did not seem to work.